### DISCUSSION TITLE
NPM依赖修改版本有时候可以出来列表,有时候出不来

### DISCUSSION THREAD

**Discussion by nightwhite** [labring/laf#676] [新春贺岁版反馈专区]
![image](https://user-images.githubusercontent.com/22722256/214098999-0780905b-eb79-4779-9ef9-4470c297cab7.png)

**Comment by LeezQ** [comment]
请求的时候，需要加个 loading 状态

**Reply by nightwhite** [reply]
有时候点了这个 无论等多久都不会去加载
